miden/tests/integration/operations/io_ops/env_ops.rs

```diff
@@ -169,3 +169,25 @@ fn build_bar_hash() -> [u64; 4] {
         bar_hash[3].as_int(),
     ]
 }
+
+// CLK INSTRUCTION
+// ================================================================================================
+
+#[test]
+fn clk() {
+    let test = build_op_test!("clk");
+    test.expect_stack(&[1]);
+
+    let source = "
+        proc.foo
+            push.1
+            push.1
+            clk
+        end
+        begin
+            exec.foo
+        end";
+
+    let test = build_test!(source, &[]);
+    test.expect_stack(&[5, 1, 1]);
```

Comment:
I'm curious why it is 5 here? I would have thought that it would be 4.

Reply:
I too was wondering that it should be 4 instead of 5. I had this sequence in my mind:

```
span
pad
incr
pad
incr
clk
```

When I cross-checked this in the VM, it's executing a `None` @ 0. Hence, it's coming out to be 5.